### PR TITLE
Fix drifting times  Issue #228

### PR DESCRIPTION
**Pull request by jtmcdole**

### CODE REVIEW

**Comment by jtmcdole** [comment]
Not sure why this didn't link to #228

**Comment by jtmcdole** [comment]
ping

**Comment by yjbanov** [comment]
Just to be paranoid, you made sure the new tests fail without the fix? If yes, LGTM.

File: lib/src/async/metronome.dart [hunk 1]
```diff
@@ -98,6 +98,6 @@ class Metronome extends Stream<DateTime> {
   _tickDate() {
     DateTime now = clock.now();
     _controller.add(now);
```
Comment (justinfagnani) [inline lib/src/async/metronome.dart]: Inline the local `now` and add a comment so someone like me doesn't wonder wtf is going on here :)

Comment (jtmcdole) [inline lib/src/async/metronome.dart]: Will do.

**Comment by jtmcdole** [comment]
@yjbanov, yep. They blew up exactly how I knew they would.

**Comment by cbracken** [comment]
lgtm

**Comment by cbracken** [comment]
Merged as e47cee3

**Pull request closed without merging**